### PR TITLE
add README for backend

### LINKED ISSUE #4: README.md - back-end

### PR DESCRIPTION
**Pull request by antoni909**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+## Install
+
+```sh
+npm install
+```
+
+## Author
+
+* Website: https://github.com/antoni909
```
Comment (MuckT) [inline README.md]: Update to authors(s) and include all project members.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```
Comment (MuckT) [inline package.json]: Can this be a dev dependency instead of dependency?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
...
+
+</p>
+
+> backend server for the healthy-queue application
+
+### 🏠 [Homepage](https://github.com/healthy-queue)
+
+### ✨ [Demo](#blank)
+
+## Install
```
Comment (MuckT) [inline README.md]: Add run steps.

**Review by MuckT — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (antoni909) [reply README.md]: all authors added

File: package.json (hunk 2 above)
Comment (antoni909) [reply package.json]: moved to dev dep

File: README.md (hunk 3 above)
Comment (antoni909) [reply README.md]: added basic usage instructions

**Review by ayrat-gimranov — APPROVED** [review]

**Review by MuckT — APPROVED** [review]
:shipit:

**Pull request merged**